homeassistant/components/zha/core/registries.py

```diff
@@ -196,6 +196,30 @@ class MatchRule:
         factory=frozenset, converter=set_or_callable
     )
 
+    @property
+    def weight(self) -> int:
+        """Return the weight of the matching rule.
+
+        Most specific matches should be preferred over less specific. Model matching
+        rules have a priority over manufacturer matching rules and rules matching a
+        single model/manufacturer get a better priority over rules matching multiple
+        models/manufacturers. And any model or manufacturers matching rules get better
+        priority over rules matching only channels.
+        But in case of a channel name/channel id matching, we give rules matching
+        multiple channels a better prioirty over rules matching a single channel.
```

Comment:
priority

Reply:
Doh.
So I don't think codespell checks comments
```
(hass) lex@debba:~/lex/src/ac-hass$ git commit -m "Add comment."
black....................................................................Passed
codespell................................................................Passed
flake8...................................................................Passed
bandit...................................................................Passed
isort....................................................................Passed
Check JSON...........................................(no files to check)Skipped
Don't commit to branch...................................................Passed
mypy.....................................................................Passed
gen_requirements_all.................................(no files to check)Skipped
hassfest.............................................(no files to check)Skipped
```

Reply:
Fixed in https://github.com/home-assistant/core/pull/33017/commits/63ec4a58d510a2bac4f5f8c25308d28588f251a3